src/test/app/forms/validation/validators/atLeastOnePopulatedRow.ts

```diff
@@ -0,0 +1,62 @@
+import { expect } from 'chai'
+
+import { AtLeastOnePopulatedRowConstraint } from 'forms/validation/validators/atLeastOnePopulatedRow'
+
+const EMPTY: boolean = true
+const POPULATED: boolean = false
+
+describe('AtLeastOnePopulatedRowConstraint', () => {
+  const constraint: AtLeastOnePopulatedRowConstraint = new AtLeastOnePopulatedRowConstraint()
+
+  describe('validate', () => {
+
+    context('should accept', () => {
+
+      it('should accept null value', () => {
+        expect(constraint.validate(null)).to.be.equal(true)
```

Comment:
null?

Reply:
as below. We can try to change it in one PR in all validators at once.